include/flamegpu/defines.h

```diff
@@ -17,6 +17,10 @@ constexpr const char* ID_VARIABLE_NAME = "_id";
  * If this value is changed, things may break
  */
 constexpr id_t ID_NOT_SET = 0;
+/**
+* Typedef for verbosity level of the API
+*/
+typedef enum {QUIET, DEFAULT, VERBOSE} verbosity;
```

Comment:
From the docs
```
ensemble.Config().error_level = pyflamegpu.CUDAEnsembleConfig.Fast
```
`CUDAEnsembleConfig` is the name of the class which nests the enum, not the name of the enum's class.

Reply:
It's defined here, just `enum`, rather than `class enum`, not sure of the difference.
https://github.com/FLAMEGPU/FLAMEGPU2/blob/0f5a4c597560a6991a9a9b33c730fb9fc18622ad/include/flamegpu/gpu/CUDAEnsemble.h#L61

Reply:
Have used a scoped enum `Verbosity`. Making this `Verbisty_t` will look ugly in SWIG so have encured that it has a fully qualified type name whereever one is declared. E.g. `flamegpu::Verbosity verbosity;`.

The enums have also been renamed to use Title Case this produces nicer SWIG bindings. E.g. `pyflamegpu.Verbosity_Quiet` (or `flamegpu::Verbosity::Quiet` in c++).

This also addresses 2 of robs comments.